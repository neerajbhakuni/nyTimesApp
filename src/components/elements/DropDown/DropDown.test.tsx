import { render } from "@testing-library/react";
import DropDown from "./index";

const mockProps = {
    selectedValue: "1",
    dayList: ["1", "2"],
    onDropDownChangeValue: jest.fn()
};

describe("rendering DropDown screen", () => {
    test("Shows DropDown", () => {
        const { container } = render(<DropDown {...mockProps} />);
        expect(container).toMatchSnapshot();
    });
});
