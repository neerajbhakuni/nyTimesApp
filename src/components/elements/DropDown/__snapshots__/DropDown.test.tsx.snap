// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering DropDown screen Shows DropDown 1`] = `
<div>
  <div
    class="flex justify-end mr-10"
  >
    <div
      class="text-2xl font-bold text-gray-800 pr-8"
    >
      Select Filter:
      <select
        class="text-2xl font-bold text-gray-800 ml-4 px-8 text-center border-solid border-2 border-black rounded"
      >
        <option
          class="text-2xl text-gray-800 text-left"
          value="1"
        >
          1 day
        </option>
        <option
          class="text-2xl text-gray-800 text-left"
          value="2"
        >
          2 days
        </option>
      </select>
    </div>
  </div>
</div>
`;
